lib/c-api/README.md

```diff
@@ -148,48 +148,48 @@ $ make test-capi
 
 `wasmer config` output various configuration information needed to compile programs which use Wasmer.
 
-### `wasmer config --pkg-config`
+### `wasmer config get pkg-config`
```

Comment:
We shouldn't change this in any case

Reply:
Reverted in https://github.com/wasmerio/wasmer/pull/3428/commits/710b8242f1f770271c5ab0a132f2e152047cb7dc